internal/exports/aws_s3_writer.go

```diff
@@ -52,15 +52,11 @@ func AWSS3CallbackFactory(
 		if err != nil {
 			log.Fatalf("Could not find %v: %v", awsDescriptionPath, err)
 		}
-		wg.Add(1)
-		go func() {
-			key := fmt.Sprintf("ABOUT%v", filepath.Ext(awsDescriptionPath))
-			if project != "" {
-				key = fmt.Sprintf("%v/%v", project, key)
-			}
-			upload(uploader, key, awsDescription)
-			wg.Done()
-		}()
+		key := fmt.Sprintf("ABOUT%v", filepath.Ext(awsDescriptionPath))
+		if project != "" {

```

Comment:
would it not have worked with `defer wg.Done()` inside the go-routine instead? But sure it's probably small and wont matter much.

Reply:
No, that was not enough.

Reply:
Do you have any other suggestions?

Reply:
sorry no, but it's fine to have it sequentially upload it, I think.

Reply:
Cool. I'll see what @joakimmoller thinks before resolving this.